packages/api/src/lib/comments-api.ts

```diff
@@ -163,3 +164,26 @@ export async function updateCompetencyIndicatorStatus(
     }
   );
 }
+
+export async function signUpUser(userDetail: UserSignUpDetail) {
+  return await axios.post('/api/signup', { userDetail });
+}
+
+export async function logInUser(email: string, password: string) {
+  try {
+    const response = await axios.post('/api/login', { email, password });
+    return response;
+  } catch (error: any) {
+    return error.response;
+  }
+}
+
+export async function verifyEmail(email: string, verificationToken: string) {
+  return await axios.get('/api/verify-email', {
+    params: { email, verificationToken },
+  });
+}
+
+export async function forgotEmail(email: string) {
```

Comment:
rename function to `sendForgotEmail` and API to `send-forgot-email`.

The first word of functions and APIs should be verbs. eg.  `check`, `verify`, `send`, `add`, `update`, `delete` etc.

This helps you understand what the function/API will do.

`forgotEmail` also has a verb: `forget` but that is not the action that the API will take.